react/react1/week2/todo-list/src/Header.js

```diff
@@ -0,0 +1,9 @@
+const Header = () => {
+  return (
+    <header>
+      <h1>Todo list</h1>
+    </header>
+  );
+};
+
```

Comment:
For this to be a dynamic component, it could accept at least the ``title`` as prop and then render that accordingly 👍🏽

Reply:
Made title as a prop in Header.